media/css/cms/flare-notification.css

```diff
@@ -125,57 +142,11 @@
     font-size: 14px;
 }
 
-.notification-blue {
-    background: light-dark(var(--blue-light-blue), var(--blue-medium-blue));
-}
-
-.notification-purple {
-    background: light-dark(var(--purple-light-purple), var(--purple-purple));
-}
-
-.notification-orange {
-    background: light-dark(var(--orange-light-orange), var(--orange-medium-orange));
-}
-
-.notification-yellow {
-    background: var(--yellow-yellow);
-}
-
-.notification-black {
-    background: var(--neutrals-charcoal);
-}
-
-.notification-white {
-    background: light-dark(var(--neutrals-white), var(--neutrals-ash));
-}
-
 .notification-outline {
```

Comment:
The tests are complaining because you defined this twice. Possible fixes: move the border declaration up to line 91. Add a comment ignoring this rule.

I'm not usually a fan of ignoring rules but sometimes it makes sense to because it make the file more human readable but I think in this case grouping them together on line 91 makes sense.